src/quantum/README.rst

```diff
@@ -0,0 +1,188 @@
+=======================================
+Microsoft Azure CLI 'quantum' Extension
+=======================================
+
+Azure Quantum is the first open Quantum computing platform. It offers a range of services
+from quantum hardware to full-state simulators and quantum inspired optimizations,
+providing developers and customers access to the most competitive quantum offering
+on the market.
+
+To learn more about azure quantum, visit:
+https://azure.microsoft.com/en-us/services/quantum/
+
+To learn more about quantum computing and Microsoft's Quantum Development Kit, visit:
+https://docs.microsoft.com/quantum/
+
+
+Using the `az quantum` extension to list and manage jobs in Azure Quantum
+=========================================================================
+
+1. Log in to Azure using your credentials.
+
+   .. code-block:: 
+
+      az login
+
+   .. note::
+      In case you have more than one subscription associated with your Azure account you must specify the 
+      subscription you want to use. You can do this with the command `az account set -s <Your subscription ID>`.
+
+
+2. Install the Quantum extension for the Azure CLI
+
+   .. code-block::
+
+      az extension add --name quantum
+
+
+3. You can see all the Azure Quantum workspaces in your subscription with the `az quantum workspace list` command:
```

Comment:
How to create a workspace so that users can see workspaces?

Reply:
I added a note on step 3 about this. At this time, the user needs to create the quantum workspaces in the Azure Portal in order to use them for Quantum Computing. Most of our users will have created their environment using the Azure Portal before using the CLI, but just in case, I added a message in the `create` command, and more explicit instructions.

Next month, we'll release an update to the CLI that configures workspaces fully from the extension, and at that time, we'll instruct users to create their workspaces this way too.

Reply:
Thanks, my confusion is resolved. Look forward to CLI support of "create workspace" in later releases.